lib/libutee/include/pta_imx_manufacturing_protection.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2019. 2023 NXP
```

Comment:
`2018-2023`?

Reply:
Nope, between 2019 and 2022, there was no change.